### DISCUSSION TITLE
Drag/drop account sorting & account sections

### DISCUSSION THREAD

**Discussion by mrconstantine111** [ghostfolio/ghostfolio#2718] [Ideas]
First off, a great many thanks to the developers behind this awesome project.
As I understand the evolving optimization of its functionality is the aim, I wish to contribute by suggesting the most straightforward UI optimizations I feel this application to be deserving of:

- In Accounts: Drag/drop custom sorting of accounts (+ memory retention of custom sorting)
- In Accounts: Create custom account sections (acting like separators in the account list)
- In Accounts: Show/edit portfolio holdings inside each account as indented sub-listings on the main Accounts view (perhaps show/hide dropdown toggle for holdings inside account)
- In Activities (view per account): Show separate columns for 'initial value', 'gain/loss', 'current value'
- In Activities (view per account): Either allow custom positioning of columns, or shift column sequence to: 'account', 'name', 'date', 'type', 'quantity', 'price', 'fee', initial value', 'gain/loss', 'current value'
- In Settings: select starting view when entering the app (initiating by account oversight may be preferrable to some)

**Comment by dtslvr** [comment]
Hi @mrconstantine111

Thank you very much for your thoughtful feedback and kind words about this project!

While we are committed to keeping Ghostfolio lightweight, we understand the importance of more advanced features. Among your suggestions, I find the addition of holdings to the account detail dialog particularly interesting.

If you're keen on contributing to this feature, I'd be happy to create a ticket for it with more details. Let me know if you're interested.

**Reply by mrconstantine111** [reply]
Hi Thomas

Likewise, thank you for the kind reception of my feedback and for inviting me to contribute to this feature.
I must clarify that my coding experience is limited and I do not consider myself a developer.
My fortes are rooted in efficiency & logic, with a keen eye for detail towards optimization of functionality.

If you consider the above a useful contribution, I would be happy to.

Sincerely,
Nicholas
